tools/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -23,19 +23,19 @@ import com.nvidia.spark.rapids.tool.qualification.PluginTypeChecker
 import org.apache.spark.internal.Logging
 import org.apache.spark.sql.execution.SparkPlanInfo
 import org.apache.spark.sql.execution.ui.{SparkPlanGraph, SparkPlanGraphCluster, SparkPlanGraphNode}
-import org.apache.spark.sql.rapids.tool.AppBase
+import org.apache.spark.sql.rapids.tool.{AppBase, ToolUtils}
 
-case class ExecInfo(
-    sqlID: Long,
-    exec: String,
-    expr: String,
-    speedupFactor: Double,
-    duration: Option[Long],
-    nodeId: Long,
-    isSupported: Boolean,
-    children: Option[Seq[ExecInfo]], // only one level deep
-    stages: Seq[Int] = Seq.empty,
-    shouldRemove: Boolean = false) {
+class ExecInfo(
```

Comment:
Just for my understanding - Why was this changed to `class` from `case class` ?

Reply:
Spark gave circular dependency error  when used in Spark itself on it since it uses ExecInfo for children. No problems when run with the tool